common/utilities/os/hresult.h

```diff
@@ -7,7 +7,7 @@
 #include <functional>
 #include <comdef.h>
 #include <sstream>
-#include "../string/string-utilities.h"
+#include "../string/windows/windows.h"
```

Comment:
Why a whole folder for Windows?

Reply:
I mean, if you make a whole folder, then the header name needs to be more specific, no?

Reply:
done